2020/08/13 16:35:22 [error] 27#27: *7 connect() failed (111: Connection refused) while connecting to upstream, client: 172.21.0.1, server: _, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "127.0.0.1:83"
2020/08/13 16:35:22 [error] 27#27: *7 connect() failed (111: Connection refused) while connecting to upstream, client: 172.21.0.1, server: _, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "127.0.0.1:83", referrer: "http://127.0.0.1:83/"
2020/08/13 16:35:22 [error] 27#27: *7 connect() failed (111: Connection refused) while connecting to upstream, client: 172.21.0.1, server: _, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "127.0.0.1:83"
2020/08/13 16:35:22 [error] 27#27: *7 connect() failed (111: Connection refused) while connecting to upstream, client: 172.21.0.1, server: _, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "127.0.0.1:83", referrer: "http://127.0.0.1:83/"
2020/08/13 16:37:25 [emerg] 1#1: upstream "php" may not have port 9000 in /etc/nginx/conf.d/default.conf:17
2020/08/13 16:37:26 [emerg] 1#1: upstream "php" may not have port 9000 in /etc/nginx/conf.d/default.conf:17
2020/08/13 16:37:27 [emerg] 1#1: upstream "php" may not have port 9000 in /etc/nginx/conf.d/default.conf:17
2020/08/13 16:37:27 [emerg] 1#1: upstream "php" may not have port 9000 in /etc/nginx/conf.d/default.conf:17
2020/08/13 16:37:29 [emerg] 1#1: upstream "php" may not have port 9000 in /etc/nginx/conf.d/default.conf:17
2020/08/13 16:37:31 [emerg] 1#1: upstream "php" may not have port 9000 in /etc/nginx/conf.d/default.conf:17
2020/08/13 16:37:35 [emerg] 1#1: upstream "php" may not have port 9000 in /etc/nginx/conf.d/default.conf:17
2020/08/13 16:37:42 [emerg] 1#1: upstream "php" may not have port 9000 in /etc/nginx/conf.d/default.conf:17
2020/08/13 16:37:56 [error] 27#27: *2 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.22.0.1, server: _, request: "GET / HTTP/1.1", upstream: "fastcgi://172.22.0.2:9000", host: "127.0.0.1:83"
2020/08/13 16:37:56 [error] 27#27: *2 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.22.0.1, server: _, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.22.0.2:9000", host: "127.0.0.1:83", referrer: "http://127.0.0.1:83/"
2020/08/13 16:48:11 [error] 27#27: *2 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.20.0.1, server: _, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.2:9000", host: "127.0.0.1:83"
2020/08/13 16:48:11 [error] 27#27: *2 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.20.0.1, server: _, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.20.0.2:9000", host: "127.0.0.1:83", referrer: "http://127.0.0.1:83/"
2020/09/03 16:17:25 [error] 27#27: *1 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.20.0.1, server: _, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1:82"
2020/09/03 16:22:44 [error] 27#27: *3 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.20.0.1, server: _, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1:82"
2020/09/03 16:22:45 [error] 27#27: *5 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.20.0.1, server: _, request: "GET /_debugbar/assets/javascript?v=1599150027 HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1:82", referrer: "http://127.0.0.1:82/"
2020/09/03 16:22:45 [error] 27#27: *3 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.20.0.1, server: _, request: "GET /_debugbar/assets/stylesheets?v=1599150027 HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1:82", referrer: "http://127.0.0.1:82/"
2020/09/03 16:24:29 [error] 27#27: *8 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.20.0.1, server: _, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1:82"
2020/11/22 11:20:04 [error] 28#28: *1 FastCGI sent in stderr: "Primary script unknown" while reading response header from upstream, client: 172.20.0.1, server: 127.0.0.1, request: "GET /_intellij_phpdebug_validator.php HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1:82"
2020/11/22 11:42:17 [error] 27#27: *1 FastCGI sent in stderr: "Primary script unknown" while reading response header from upstream, client: 172.21.0.1, server: 127.0.0.1, request: "GET /_intellij_phpdebug_validator.php HTTP/1.1", upstream: "fastcgi://172.21.0.3:9000", host: "127.0.0.1:82"
2020/11/22 11:42:20 [error] 27#27: *3 FastCGI sent in stderr: "Primary script unknown" while reading response header from upstream, client: 172.21.0.1, server: 127.0.0.1, request: "GET /_intellij_phpdebug_validator.php HTTP/1.1", upstream: "fastcgi://172.21.0.3:9000", host: "127.0.0.1:82"
2020/11/22 11:42:31 [error] 27#27: *5 FastCGI sent in stderr: "Primary script unknown" while reading response header from upstream, client: 172.21.0.1, server: 127.0.0.1, request: "GET /_intellij_phpdebug_validator.php HTTP/1.1", upstream: "fastcgi://172.21.0.3:9000", host: "127.0.0.1:82"
2020/11/22 11:45:51 [error] 27#27: *7 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.21.0.1, server: 127.0.0.1, request: "GET /_intellij_phpdebug_validator.php HTTP/1.1", upstream: "fastcgi://172.21.0.3:9000", host: "127.0.0.1:82"
2020/11/22 11:46:14 [error] 27#27: *9 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.21.0.1, server: 127.0.0.1, request: "GET /_intellij_phpdebug_validator.php HTTP/1.1", upstream: "fastcgi://172.21.0.3:9000", host: "127.0.0.1:82"
2020/11/22 11:48:25 [error] 27#27: *1 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.22.0.1, server: 127.0.0.1, request: "GET /_intellij_phpdebug_validator.php HTTP/1.1", upstream: "fastcgi://172.22.0.2:9000", host: "127.0.0.1:82"
2020/11/22 11:49:03 [error] 27#27: *1 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.23.0.1, server: 127.0.0.1, request: "GET /_intellij_phpdebug_validator.php HTTP/1.1", upstream: "fastcgi://172.23.0.3:9000", host: "127.0.0.1:82"
2020/11/22 11:49:25 [error] 27#27: *3 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.23.0.1, server: 127.0.0.1, request: "GET /_intellij_phpdebug_validator.php HTTP/1.1", upstream: "fastcgi://172.23.0.3:9000", host: "127.0.0.1:82"
2020/11/22 11:49:30 [error] 27#27: *5 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.23.0.1, server: 127.0.0.1, request: "GET /_intellij_phpdebug_validator.php HTTP/1.1", upstream: "fastcgi://172.23.0.3:9000", host: "127.0.0.1:82"
2020/11/22 11:51:14 [error] 27#27: *7 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.23.0.1, server: 127.0.0.1, request: "GET /_intellij_phpdebug_validator.php HTTP/1.1", upstream: "fastcgi://172.23.0.3:9000", host: "127.0.0.1:82"
2020/11/22 11:51:16 [error] 27#27: *7 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.23.0.1, server: 127.0.0.1, request: "GET /_intellij_phpdebug_validator.php HTTP/1.1", upstream: "fastcgi://172.23.0.3:9000", host: "127.0.0.1:82"
2020/11/22 11:51:16 [error] 27#27: *7 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.23.0.1, server: 127.0.0.1, request: "GET /_intellij_phpdebug_validator.php HTTP/1.1", upstream: "fastcgi://172.23.0.3:9000", host: "127.0.0.1:82"
2020/11/22 11:51:19 [error] 27#27: *7 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.23.0.1, server: 127.0.0.1, request: "GET /_intellij_phpdebug_validator.php HTTP/1.1", upstream: "fastcgi://172.23.0.3:9000", host: "127.0.0.1:82"
2020/11/22 11:51:19 [error] 27#27: *7 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.23.0.1, server: 127.0.0.1, request: "GET /_intellij_phpdebug_validator.php HTTP/1.1", upstream: "fastcgi://172.23.0.3:9000", host: "127.0.0.1:82"
2020/11/22 11:52:23 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.24.0.1, server: 127.0.0.1, request: "GET /_intellij_phpdebug_validator.php HTTP/1.1", upstream: "fastcgi://172.24.0.2:9000", host: "127.0.0.1:82"
2020/11/22 11:53:46 [error] 27#27: *1 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.26.0.1, server: 127.0.0.1, request: "GET /_intellij_phpdebug_validator.php HTTP/1.1", upstream: "fastcgi://172.26.0.3:9000", host: "127.0.0.1:82"
2020/11/22 11:53:55 [error] 27#27: *3 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.26.0.1, server: 127.0.0.1, request: "GET /?XDEBUG_SESSION_START=10259 HTTP/1.1", upstream: "fastcgi://172.26.0.3:9000", host: "127.0.0.1:82"
2020/11/22 11:54:01 [error] 27#27: *3 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.26.0.1, server: 127.0.0.1, request: "GET /?XDEBUG_SESSION_START=12959 HTTP/1.1", upstream: "fastcgi://172.26.0.3:9000", host: "127.0.0.1:82"
2020/11/22 11:54:11 [error] 27#27: *3 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.26.0.1, server: 127.0.0.1, request: "GET /?XDEBUG_SESSION_START=12959 HTTP/1.1", upstream: "fastcgi://172.26.0.3:9000", host: "127.0.0.1:82"
2020/11/22 11:54:11 [error] 27#27: *3 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.26.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.26.0.3:9000", host: "127.0.0.1:82", referrer: "http://127.0.0.1:82/?XDEBUG_SESSION_START=12959"
2020/11/22 11:54:15 [error] 27#27: *3 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.26.0.1, server: 127.0.0.1, request: "GET /?XDEBUG_SESSION_START=12959 HTTP/1.1", upstream: "fastcgi://172.26.0.3:9000", host: "127.0.0.1:82"
2020/11/22 11:54:15 [error] 27#27: *3 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.26.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.26.0.3:9000", host: "127.0.0.1:82", referrer: "http://127.0.0.1:82/?XDEBUG_SESSION_START=12959"
2020/11/22 11:54:16 [error] 27#27: *3 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.26.0.1, server: 127.0.0.1, request: "GET /?XDEBUG_SESSION_START=12959 HTTP/1.1", upstream: "fastcgi://172.26.0.3:9000", host: "127.0.0.1:82"
2020/11/22 11:54:16 [error] 27#27: *3 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.26.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.26.0.3:9000", host: "127.0.0.1:82", referrer: "http://127.0.0.1:82/?XDEBUG_SESSION_START=12959"
2020/11/22 11:54:17 [error] 27#27: *3 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.26.0.1, server: 127.0.0.1, request: "GET /?XDEBUG_SESSION_START=12959 HTTP/1.1", upstream: "fastcgi://172.26.0.3:9000", host: "127.0.0.1:82"
2020/11/22 11:54:17 [error] 27#27: *3 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.26.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.26.0.3:9000", host: "127.0.0.1:82", referrer: "http://127.0.0.1:82/?XDEBUG_SESSION_START=12959"
2020/11/22 11:54:17 [error] 27#27: *3 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.26.0.1, server: 127.0.0.1, request: "GET /?XDEBUG_SESSION_START=12959 HTTP/1.1", upstream: "fastcgi://172.26.0.3:9000", host: "127.0.0.1:82"
2020/11/22 11:54:17 [error] 27#27: *3 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.26.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.26.0.3:9000", host: "127.0.0.1:82", referrer: "http://127.0.0.1:82/?XDEBUG_SESSION_START=12959"
2020/11/22 11:54:18 [error] 27#27: *3 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.26.0.1, server: 127.0.0.1, request: "GET /?XDEBUG_SESSION_START=12959 HTTP/1.1", upstream: "fastcgi://172.26.0.3:9000", host: "127.0.0.1:82"
2020/11/22 11:54:18 [error] 27#27: *3 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.26.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.26.0.3:9000", host: "127.0.0.1:82", referrer: "http://127.0.0.1:82/?XDEBUG_SESSION_START=12959"
2020/11/22 11:54:48 [error] 27#27: *3 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.26.0.1, server: 127.0.0.1, request: "GET /?XDEBUG_SESSION_START=14944 HTTP/1.1", upstream: "fastcgi://172.26.0.3:9000", host: "127.0.0.1:82"
2020/11/22 11:54:48 [error] 27#27: *3 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.26.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.26.0.3:9000", host: "127.0.0.1:82", referrer: "http://127.0.0.1:82/?XDEBUG_SESSION_START=14944"
2020/11/22 11:57:05 [error] 27#27: *22 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.26.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://172.26.0.3:9000", host: "localhost:82"
2020/11/22 11:57:05 [error] 27#27: *22 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.26.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.26.0.3:9000", host: "localhost:82", referrer: "http://localhost:82/"
2020/11/22 12:00:35 [error] 27#27: *2 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.27.0.1, server: 127.0.0.1, request: "GET /?XDEBUG_SESSION_START=11970 HTTP/1.1", upstream: "fastcgi://172.27.0.2:9000", host: "127.0.0.1:82"
2020/11/22 12:00:35 [error] 27#27: *2 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.27.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.27.0.2:9000", host: "127.0.0.1:82", referrer: "http://127.0.0.1:82/?XDEBUG_SESSION_START=11970"
2020/11/22 12:14:08 [error] 27#27: *2 upstream timed out (110: Connection timed out) while reading response header from upstream, client: 172.29.0.1, server: 127.0.0.1, request: "GET /?XDEBUG_SESSION_START=10539 HTTP/1.1", upstream: "fastcgi://172.29.0.2:9000", host: "127.0.0.1:82"
2020/11/22 12:15:08 [error] 27#27: *2 upstream timed out (110: Connection timed out) while reading response header from upstream, client: 172.29.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.29.0.2:9000", host: "127.0.0.1:82", referrer: "http://127.0.0.1:82/?XDEBUG_SESSION_START=10539"
2020/11/22 12:15:49 [error] 27#27: *5 upstream timed out (110: Connection timed out) while reading response header from upstream, client: 172.29.0.1, server: 127.0.0.1, request: "GET /?XDEBUG_SESSION_START=18409 HTTP/1.1", upstream: "fastcgi://172.29.0.2:9000", host: "127.0.0.1:82"
2020/11/22 12:15:53 [error] 27#27: *7 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.29.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.29.0.2:9000", host: "127.0.0.1:82", referrer: "http://127.0.0.1:82/?XDEBUG_SESSION_START=18409"
2020/11/22 12:16:01 [error] 27#27: *7 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.29.0.1, server: 127.0.0.1, request: "GET /?XDEBUG_SESSION_START=18246 HTTP/1.1", upstream: "fastcgi://172.29.0.2:9000", host: "127.0.0.1:82"
2020/11/22 12:16:02 [error] 27#27: *7 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.29.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.29.0.2:9000", host: "127.0.0.1:82", referrer: "http://127.0.0.1:82/?XDEBUG_SESSION_START=18246"
2020/11/22 12:16:14 [error] 27#27: *7 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.29.0.1, server: 127.0.0.1, request: "GET /?XDEBUG_SESSION_START=18421 HTTP/1.1", upstream: "fastcgi://172.29.0.2:9000", host: "127.0.0.1:82"
2020/11/22 12:16:14 [error] 27#27: *7 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.29.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.29.0.2:9000", host: "127.0.0.1:82", referrer: "http://127.0.0.1:82/?XDEBUG_SESSION_START=18421"
2020/11/22 12:16:39 [error] 27#27: *7 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.29.0.1, server: 127.0.0.1, request: "GET /?XDEBUG_SESSION_START=13614 HTTP/1.1", upstream: "fastcgi://172.29.0.2:9000", host: "127.0.0.1:82"
2020/11/22 12:16:40 [error] 27#27: *7 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.29.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.29.0.2:9000", host: "127.0.0.1:82", referrer: "http://127.0.0.1:82/?XDEBUG_SESSION_START=13614"
2020/11/22 12:17:15 [error] 27#27: *7 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.29.0.1, server: 127.0.0.1, request: "GET /?XDEBUG_SESSION_START=13288 HTTP/1.1", upstream: "fastcgi://172.29.0.2:9000", host: "127.0.0.1:82"
2020/11/22 12:17:16 [error] 27#27: *7 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.29.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.29.0.2:9000", host: "127.0.0.1:82", referrer: "http://127.0.0.1:82/?XDEBUG_SESSION_START=13288"
2020/11/22 12:53:59 [error] 27#27: *1 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.80.1, server: 127.0.0.1, request: "GET /_intellij_phpdebug_validator.php HTTP/1.1", upstream: "fastcgi://192.168.80.3:9000", host: "127.0.0.1:82"
2020/11/22 12:54:20 [error] 27#27: *3 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.80.1, server: 127.0.0.1, request: "GET /_intellij_phpdebug_validator.php HTTP/1.1", upstream: "fastcgi://192.168.80.3:9000", host: "127.0.0.1:82"
2020/11/22 12:54:26 [error] 27#27: *6 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.80.1, server: 127.0.0.1, request: "GET /?XDEBUG_SESSION_START=11009 HTTP/1.1", upstream: "fastcgi://192.168.80.3:9000", host: "127.0.0.1:82"
2020/11/22 12:55:16 [error] 27#27: *1 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.96.1, server: 127.0.0.1, request: "GET /_intellij_phpdebug_validator.php HTTP/1.1", upstream: "fastcgi://192.168.96.3:9000", host: "127.0.0.1:82"
2020/11/22 12:55:21 [error] 27#27: *4 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.96.1, server: 127.0.0.1, request: "GET /?XDEBUG_SESSION_START=18208 HTTP/1.1", upstream: "fastcgi://192.168.96.3:9000", host: "127.0.0.1:82"
2020/11/22 12:55:33 [error] 27#27: *4 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.96.1, server: 127.0.0.1, request: "GET /?XDEBUG_SESSION_START=12648 HTTP/1.1", upstream: "fastcgi://192.168.96.3:9000", host: "127.0.0.1:82"
2020/11/22 12:55:54 [error] 27#27: *4 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.96.1, server: 127.0.0.1, request: "GET /?XDEBUG_SESSION_START=19838 HTTP/1.1", upstream: "fastcgi://192.168.96.3:9000", host: "127.0.0.1:82"
2020/11/22 12:56:34 [error] 27#27: *2 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.112.1, server: 127.0.0.1, request: "GET /?XDEBUG_SESSION_START=17794 HTTP/1.1", upstream: "fastcgi://192.168.112.3:9000", host: "127.0.0.1:82"
2020/11/22 12:56:39 [error] 27#27: *4 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.112.1, server: 127.0.0.1, request: "GET /?XDEBUG_SESSION_START=14509 HTTP/1.1", upstream: "fastcgi://192.168.112.3:9000", host: "127.0.0.1:82"
2020/11/22 12:56:48 [error] 27#27: *7 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.112.1, server: 127.0.0.1, request: "GET /_intellij_phpdebug_validator.php HTTP/1.1", upstream: "fastcgi://192.168.112.3:9000", host: "127.0.0.1:82"
2020/11/22 12:57:18 [error] 27#27: *4 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.112.1, server: 127.0.0.1, request: "GET /?XDEBUG_SESSION_START=12668 HTTP/1.1", upstream: "fastcgi://192.168.112.3:9000", host: "127.0.0.1:82"
2020/11/22 12:57:22 [error] 27#27: *4 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.112.1, server: 127.0.0.1, request: "GET /?XDEBUG_SESSION_START=17794 HTTP/1.1", upstream: "fastcgi://192.168.112.3:9000", host: "127.0.0.1:82"
2020/11/22 12:57:22 [error] 27#27: *4 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.112.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://192.168.112.3:9000", host: "127.0.0.1:82", referrer: "http://127.0.0.1:82/?XDEBUG_SESSION_START=17794"
2020/11/22 12:57:26 [error] 27#27: *4 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.112.1, server: 127.0.0.1, request: "GET /?XDEBUG_SESSION_START=17794 HTTP/1.1", upstream: "fastcgi://192.168.112.3:9000", host: "127.0.0.1:82"
2020/11/22 12:57:26 [error] 27#27: *4 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.112.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://192.168.112.3:9000", host: "127.0.0.1:82", referrer: "http://127.0.0.1:82/?XDEBUG_SESSION_START=17794"
2020/11/22 12:57:27 [error] 27#27: *4 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.112.1, server: 127.0.0.1, request: "GET /?XDEBUG_SESSION_START=17794 HTTP/1.1", upstream: "fastcgi://192.168.112.3:9000", host: "127.0.0.1:82"
2020/11/22 12:57:27 [error] 27#27: *4 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.112.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://192.168.112.3:9000", host: "127.0.0.1:82", referrer: "http://127.0.0.1:82/?XDEBUG_SESSION_START=17794"
2020/11/22 12:57:28 [error] 27#27: *4 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.112.1, server: 127.0.0.1, request: "GET /?XDEBUG_SESSION_START=17794 HTTP/1.1", upstream: "fastcgi://192.168.112.3:9000", host: "127.0.0.1:82"
2020/11/22 12:57:28 [error] 27#27: *4 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.112.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://192.168.112.3:9000", host: "127.0.0.1:82", referrer: "http://127.0.0.1:82/?XDEBUG_SESSION_START=17794"
2020/11/22 12:58:10 [error] 27#27: *1 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.128.1, server: 127.0.0.1, request: "GET /_intellij_phpdebug_validator.php HTTP/1.1", upstream: "fastcgi://192.168.128.2:9000", host: "127.0.0.1:82"
2020/11/22 12:58:20 [error] 27#27: *3 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.128.1, server: 127.0.0.1, request: "GET /?XDEBUG_SESSION_START=13426 HTTP/1.1", upstream: "fastcgi://192.168.128.2:9000", host: "127.0.0.1:82"
2020/11/22 12:58:20 [error] 27#27: *3 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.128.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://192.168.128.2:9000", host: "127.0.0.1:82", referrer: "http://127.0.0.1:82/?XDEBUG_SESSION_START=13426"
2020/11/22 12:59:27 [error] 27#27: *3 upstream timed out (110: Connection timed out) while reading response header from upstream, client: 192.168.128.1, server: 127.0.0.1, request: "GET /?XDEBUG_SESSION_START=10855 HTTP/1.1", upstream: "fastcgi://192.168.128.2:9000", host: "127.0.0.1:82"
2020/11/22 13:00:27 [error] 27#27: *3 upstream timed out (110: Connection timed out) while reading response header from upstream, client: 192.168.128.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://192.168.128.2:9000", host: "127.0.0.1:82", referrer: "http://127.0.0.1:82/?XDEBUG_SESSION_START=10855"
2020/11/22 13:03:46 [error] 27#27: *16 upstream timed out (110: Connection timed out) while reading response header from upstream, client: 192.168.128.1, server: 127.0.0.1, request: "GET /?XDEBUG_SESSION_START=10427 HTTP/1.1", upstream: "fastcgi://192.168.128.2:9000", host: "127.0.0.1:82"
2020/11/22 13:04:18 [error] 27#27: *1 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.144.1, server: 127.0.0.1, request: "GET /?XDEBUG_SESSION_START=13089 HTTP/1.1", upstream: "fastcgi://192.168.144.2:9000", host: "127.0.0.1:82"
2020/11/22 13:04:18 [error] 27#27: *1 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.144.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://192.168.144.2:9000", host: "127.0.0.1:82", referrer: "http://127.0.0.1:82/?XDEBUG_SESSION_START=13089"
2020/11/22 13:04:48 [error] 27#27: *1 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.144.1, server: 127.0.0.1, request: "GET /?XDEBUG_SESSION_START=17583 HTTP/1.1", upstream: "fastcgi://192.168.144.2:9000", host: "127.0.0.1:82"
2020/11/22 13:05:00 [error] 27#27: *1 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.144.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://192.168.144.2:9000", host: "127.0.0.1:82", referrer: "http://127.0.0.1:82/?XDEBUG_SESSION_START=17583"
2020/11/22 13:05:10 [error] 27#27: *1 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.144.1, server: 127.0.0.1, request: "GET /?XDEBUG_SESSION_START=15092 HTTP/1.1", upstream: "fastcgi://192.168.144.2:9000", host: "127.0.0.1:82"
2020/11/22 13:05:17 [error] 27#27: *1 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.144.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://192.168.144.2:9000", host: "127.0.0.1:82", referrer: "http://127.0.0.1:82/?XDEBUG_SESSION_START=15092"
2020/11/22 13:05:27 [error] 27#27: *1 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.144.1, server: 127.0.0.1, request: "GET /?XDEBUG_SESSION_START=19209 HTTP/1.1", upstream: "fastcgi://192.168.144.2:9000", host: "127.0.0.1:82"
2020/11/22 13:06:03 [error] 27#27: *1 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.144.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://192.168.144.2:9000", host: "127.0.0.1:82", referrer: "http://127.0.0.1:82/?XDEBUG_SESSION_START=19209"
2020/11/22 13:14:49 [error] 27#27: *1 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.160.1, server: 127.0.0.1, request: "GET /_intellij_phpdebug_validator.php HTTP/1.1", upstream: "fastcgi://192.168.160.2:9000", host: "127.0.0.1:82"
2020/11/22 13:15:13 [error] 27#27: *4 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.160.1, server: 127.0.0.1, request: "GET /?XDEBUG_SESSION_START=14920 HTTP/1.1", upstream: "fastcgi://192.168.160.2:9000", host: "127.0.0.1:82"
2020/11/22 13:15:15 [error] 27#27: *4 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.160.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://192.168.160.2:9000", host: "127.0.0.1:82", referrer: "http://127.0.0.1:82/?XDEBUG_SESSION_START=14920"
2020/11/22 13:15:28 [error] 27#27: *4 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.160.1, server: 127.0.0.1, request: "GET /?XDEBUG_SESSION_START=14760 HTTP/1.1", upstream: "fastcgi://192.168.160.2:9000", host: "127.0.0.1:82"
2020/11/22 13:15:33 [error] 27#27: *4 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.160.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://192.168.160.2:9000", host: "127.0.0.1:82", referrer: "http://127.0.0.1:82/?XDEBUG_SESSION_START=14760"
2020/11/22 13:16:05 [error] 27#27: *4 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.160.1, server: 127.0.0.1, request: "GET /?XDEBUG_SESSION_START=10390 HTTP/1.1", upstream: "fastcgi://192.168.160.2:9000", host: "127.0.0.1:82"
2020/11/22 13:16:07 [error] 27#27: *4 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.160.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://192.168.160.2:9000", host: "127.0.0.1:82", referrer: "http://127.0.0.1:82/?XDEBUG_SESSION_START=10390"
2020/11/22 13:16:17 [error] 27#27: *4 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.160.1, server: 127.0.0.1, request: "GET /?XDEBUG_SESSION_START=11703 HTTP/1.1", upstream: "fastcgi://192.168.160.2:9000", host: "127.0.0.1:82"
2020/11/22 13:16:22 [error] 27#27: *4 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.160.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://192.168.160.2:9000", host: "127.0.0.1:82", referrer: "http://127.0.0.1:82/?XDEBUG_SESSION_START=11703"
2020/11/22 13:16:43 [error] 27#27: *4 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.160.1, server: 127.0.0.1, request: "GET /?XDEBUG_SESSION_START=11624 HTTP/1.1", upstream: "fastcgi://192.168.160.2:9000", host: "127.0.0.1:82"
2020/11/22 13:17:43 [error] 27#27: *4 upstream timed out (110: Connection timed out) while reading response header from upstream, client: 192.168.160.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://192.168.160.2:9000", host: "127.0.0.1:82", referrer: "http://127.0.0.1:82/?XDEBUG_SESSION_START=11624"
2020/11/22 13:18:15 [error] 27#27: *16 upstream timed out (110: Connection timed out) while reading response header from upstream, client: 192.168.160.1, server: 127.0.0.1, request: "GET /?XDEBUG_SESSION_START=12950 HTTP/1.1", upstream: "fastcgi://192.168.160.2:9000", host: "127.0.0.1:82"
2020/11/22 13:18:15 [error] 27#27: *16 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.160.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://192.168.160.2:9000", host: "127.0.0.1:82", referrer: "http://127.0.0.1:82/?XDEBUG_SESSION_START=12950"
2020/11/22 13:18:29 [error] 27#27: *16 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.160.1, server: 127.0.0.1, request: "GET /?XDEBUG_SESSION_START=16536 HTTP/1.1", upstream: "fastcgi://192.168.160.2:9000", host: "127.0.0.1:82"
2020/11/22 13:18:29 [error] 27#27: *16 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.160.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://192.168.160.2:9000", host: "127.0.0.1:82", referrer: "http://127.0.0.1:82/?XDEBUG_SESSION_START=16536"
2020/11/22 13:19:20 [error] 27#27: *22 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.160.1, server: 127.0.0.1, request: "GET /?XDEBUG_SESSION_START=18475 HTTP/1.1", upstream: "fastcgi://192.168.160.2:9000", host: "127.0.0.1:82"
2020/11/22 13:19:20 [error] 27#27: *22 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.160.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://192.168.160.2:9000", host: "127.0.0.1:82", referrer: "http://127.0.0.1:82/?XDEBUG_SESSION_START=18475"
2020/11/22 13:22:31 [error] 27#27: *26 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.160.1, server: 127.0.0.1, request: "GET /?XDEBUG_SESSION_START=10201 HTTP/1.1", upstream: "fastcgi://192.168.160.2:9000", host: "127.0.0.1:82"
2020/11/22 13:22:35 [error] 27#27: *26 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.160.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://192.168.160.2:9000", host: "127.0.0.1:82", referrer: "http://127.0.0.1:82/?XDEBUG_SESSION_START=10201"
2020/11/22 13:25:54 [error] 27#27: *29 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.160.1, server: 127.0.0.1, request: "GET /_intellij_phpdebug_validator.php HTTP/1.1", upstream: "fastcgi://192.168.160.2:9000", host: "127.0.0.1:82"
2020/11/22 13:26:18 [error] 27#27: *31 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.160.1, server: 127.0.0.1, request: "GET /?XDEBUG_SESSION_START=17245 HTTP/1.1", upstream: "fastcgi://192.168.160.2:9000", host: "127.0.0.1:82"
2020/11/22 13:26:18 [error] 27#27: *31 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.160.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://192.168.160.2:9000", host: "127.0.0.1:82", referrer: "http://127.0.0.1:82/?XDEBUG_SESSION_START=17245"
2020/11/22 13:56:37 [error] 27#27: *1 FastCGI sent in stderr: "PHP message: XDebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.208.1, server: 127.0.0.1, request: "GET /_intellij_phpdebug_validator.php HTTP/1.1", upstream: "fastcgi://192.168.208.2:9000", host: "127.0.0.1:82"
2020/11/22 13:57:13 [error] 27#27: *3 FastCGI sent in stderr: "PHP message: XDebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.208.1, server: 127.0.0.1, request: "GET /?XDEBUG_SESSION_START=11606 HTTP/1.1", upstream: "fastcgi://192.168.208.2:9000", host: "127.0.0.1:82"
2020/11/22 13:57:13 [error] 27#27: *3 FastCGI sent in stderr: "PHP message: XDebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.208.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://192.168.208.2:9000", host: "127.0.0.1:82", referrer: "http://127.0.0.1:82/?XDEBUG_SESSION_START=11606"
2020/11/22 14:17:24 [error] 27#27: *1 FastCGI sent in stderr: "PHP message: XDebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.224.1, server: 127.0.0.1, request: "GET /_intellij_phpdebug_validator.php HTTP/1.1", upstream: "fastcgi://192.168.224.3:9000", host: "127.0.0.1:82"
2020/11/22 14:17:37 [error] 27#27: *4 FastCGI sent in stderr: "PHP message: XDebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.224.1, server: 127.0.0.1, request: "GET /?XDEBUG_SESSION_START=19571 HTTP/1.1", upstream: "fastcgi://192.168.224.3:9000", host: "127.0.0.1:82"
2020/11/22 14:17:37 [error] 27#27: *4 FastCGI sent in stderr: "PHP message: XDebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.224.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://192.168.224.3:9000", host: "127.0.0.1:82", referrer: "http://127.0.0.1:82/?XDEBUG_SESSION_START=19571"
2021/01/05 10:41:20 [error] 29#29: *2 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "loc.l:82"
2021/01/05 10:41:20 [error] 29#29: *2 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "loc.l:82", referrer: "http://loc.l:82/"
2021/01/05 10:41:40 [error] 29#29: *2 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "loc.l:82"
2021/01/05 10:41:43 [error] 29#29: *2 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "loc.l:82", referrer: "http://loc.l:82/"
2021/01/05 10:41:49 [error] 29#29: *2 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "loc.l:82"
2021/01/05 10:41:51 [error] 29#29: *2 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "loc.l:82", referrer: "http://loc.l:82/"
2021/01/05 10:41:54 [error] 29#29: *2 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "loc.l:82"
2021/01/05 10:41:55 [error] 29#29: *2 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "loc.l:82", referrer: "http://loc.l:82/"
2021/01/05 10:41:59 [error] 29#29: *2 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "loc.l:82"
2021/01/05 10:41:59 [error] 29#29: *2 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "loc.l:82", referrer: "http://loc.l:82/"
2021/01/05 10:42:02 [error] 29#29: *2 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "loc.l:82"
2021/01/05 10:42:03 [error] 29#29: *2 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "loc.l:82", referrer: "http://loc.l:82/"
2021/01/05 10:42:06 [error] 29#29: *2 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "loc.l:82"
2021/01/05 10:42:09 [error] 29#29: *2 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "loc.l:82", referrer: "http://loc.l:82/"
2021/01/05 10:42:12 [error] 29#29: *2 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "loc.l:82"
2021/01/05 10:42:13 [error] 29#29: *2 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "loc.l:82", referrer: "http://loc.l:82/"
2021/01/05 10:42:15 [error] 29#29: *2 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "loc.l:82"
2021/01/05 10:42:16 [error] 29#29: *2 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "loc.l:82", referrer: "http://loc.l:82/"
2021/01/05 10:42:20 [error] 29#29: *2 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "loc.l:82"
2021/01/05 10:42:22 [error] 29#29: *2 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "loc.l:82", referrer: "http://loc.l:82/"
2021/01/05 10:42:28 [error] 29#29: *2 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "loc.l:82"
2021/01/05 10:42:31 [error] 29#29: *2 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "loc.l:82", referrer: "http://loc.l:82/"
2021/01/05 10:42:35 [error] 29#29: *2 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "loc.l:82"
2021/01/05 10:42:37 [error] 29#29: *2 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "loc.l:82", referrer: "http://loc.l:82/"
2021/01/05 10:42:45 [error] 29#29: *2 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "loc.l:82"
2021/01/05 10:42:45 [error] 29#29: *2 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "loc.l:82", referrer: "http://loc.l:82/"
2021/01/05 10:42:48 [error] 29#29: *2 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "loc.l:82"
2021/01/05 10:42:48 [error] 29#29: *2 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "loc.l:82", referrer: "http://loc.l:82/"
2021/01/05 10:42:52 [error] 29#29: *2 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "loc.l:82"
2021/01/05 10:42:56 [error] 29#29: *2 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "loc.l:82", referrer: "http://loc.l:82/"
2021/01/05 10:42:59 [error] 29#29: *2 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "loc.l:82"
2021/01/05 10:43:06 [error] 29#29: *23 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "loc.l:82"
2021/01/05 10:43:08 [error] 29#29: *23 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "loc.l:82", referrer: "http://loc.l:82/"
2021/01/05 10:43:19 [error] 29#29: *23 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "loc.l:82"
2021/01/05 10:43:26 [error] 29#29: *23 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "loc.l:82", referrer: "http://loc.l:82/"
2021/01/05 10:43:33 [error] 29#29: *23 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "loc.l:82"
2021/01/05 10:43:35 [error] 29#29: *23 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "loc.l:82", referrer: "http://loc.l:82/"
2021/01/05 10:43:40 [error] 29#29: *23 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "loc.l:82"
2021/01/05 10:43:42 [error] 29#29: *23 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "loc.l:82", referrer: "http://loc.l:82/"
2021/01/05 10:45:25 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET /_intellij_phpdebug_validator.php HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "127.0.0.1:82"
2021/01/05 10:46:04 [error] 28#28: *3 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "loc.l:82"
2021/01/05 10:46:06 [error] 28#28: *3 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "loc.l:82", referrer: "http://loc.l:82/"
2021/01/05 10:46:37 [error] 28#28: *3 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "loc.l:82"
2021/01/05 10:46:38 [error] 28#28: *3 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "loc.l:82", referrer: "http://loc.l:82/"
2021/01/05 10:46:53 [error] 28#28: *3 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "loc.l:82"
2021/01/05 10:46:54 [error] 28#28: *3 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "loc.l:82", referrer: "http://loc.l:82/"
2021/01/05 10:47:06 [error] 28#28: *3 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "loc.l:82"
2021/01/05 10:47:13 [error] 28#28: *3 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "loc.l:82", referrer: "http://loc.l:82/"
2021/04/24 14:55:07 [error] 29#29: *1 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "loc.l:82"
2021/04/24 14:56:01 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.19.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.2:9000", host: "loc.l:82"
2021/04/24 14:56:01 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.19.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.19.0.2:9000", host: "loc.l:82", referrer: "http://loc.l:82/"
2021/04/24 14:56:01 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.19.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.2:9000", host: "loc.l:82"
2021/04/24 14:56:01 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.19.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.19.0.2:9000", host: "loc.l:82", referrer: "http://loc.l:82/"
2021/04/24 14:56:10 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.19.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.2:9000", host: "loc.l:82"
2021/04/24 14:56:10 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.19.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.19.0.2:9000", host: "loc.l:82", referrer: "http://loc.l:82/"
2021/04/24 15:07:17 [emerg] 1#1: host not found in upstream "php" in /etc/nginx/conf.d/default.conf:13
2021/04/24 15:07:17 [emerg] 1#1: host not found in upstream "php" in /etc/nginx/conf.d/default.conf:13
2021/04/24 15:07:18 [emerg] 1#1: host not found in upstream "php" in /etc/nginx/conf.d/default.conf:13
2021/04/24 15:07:18 [emerg] 1#1: host not found in upstream "php" in /etc/nginx/conf.d/default.conf:13
2021/04/24 15:07:19 [emerg] 1#1: host not found in upstream "php" in /etc/nginx/conf.d/default.conf:13
2021/04/24 15:07:21 [emerg] 1#1: host not found in upstream "php" in /etc/nginx/conf.d/default.conf:13
2021/04/24 15:07:24 [emerg] 1#1: host not found in upstream "php" in /etc/nginx/conf.d/default.conf:13
2021/04/24 15:07:31 [emerg] 1#1: host not found in upstream "php" in /etc/nginx/conf.d/default.conf:13
2021/04/24 15:07:44 [emerg] 1#1: host not found in upstream "php" in /etc/nginx/conf.d/default.conf:13
2021/04/24 15:08:10 [emerg] 1#1: host not found in upstream "php" in /etc/nginx/conf.d/default.conf:13
2021/04/24 15:08:33 [emerg] 1#1: host not found in upstream "php" in /etc/nginx/conf.d/default.conf:13
2021/04/24 15:08:33 [emerg] 1#1: host not found in upstream "php" in /etc/nginx/conf.d/default.conf:13
2021/04/24 15:08:34 [emerg] 1#1: host not found in upstream "php" in /etc/nginx/conf.d/default.conf:13
2021/04/24 15:08:34 [emerg] 1#1: host not found in upstream "php" in /etc/nginx/conf.d/default.conf:13
2021/04/24 15:08:35 [emerg] 1#1: host not found in upstream "php" in /etc/nginx/conf.d/default.conf:13
2021/04/24 15:08:37 [emerg] 1#1: host not found in upstream "php" in /etc/nginx/conf.d/default.conf:13
2021/04/24 15:10:04 [emerg] 1#1: upstream "php" may not have port 9000 in /etc/nginx/conf.d/default.conf:17
2021/04/24 15:10:04 [emerg] 1#1: upstream "php" may not have port 9000 in /etc/nginx/conf.d/default.conf:17
2021/04/24 15:10:05 [emerg] 1#1: upstream "php" may not have port 9000 in /etc/nginx/conf.d/default.conf:17
2021/04/24 15:10:05 [emerg] 1#1: upstream "php" may not have port 9000 in /etc/nginx/conf.d/default.conf:17
2021/04/24 15:10:06 [emerg] 1#1: upstream "php" may not have port 9000 in /etc/nginx/conf.d/default.conf:17
2021/04/24 15:10:08 [emerg] 1#1: upstream "php" may not have port 9000 in /etc/nginx/conf.d/default.conf:17
2021/04/24 15:10:11 [emerg] 1#1: upstream "php" may not have port 9000 in /etc/nginx/conf.d/default.conf:17
2021/04/24 15:10:18 [emerg] 1#1: upstream "php" may not have port 9000 in /etc/nginx/conf.d/default.conf:17
2021/04/24 15:11:49 [error] 28#28: *2 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.23.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://172.23.0.2:9000", host: "loc.l:82"
2021/04/24 15:11:49 [error] 28#28: *2 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.23.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.23.0.2:9000", host: "loc.l:82", referrer: "http://loc.l:82/"
2021/04/24 15:19:40 [error] 29#29: *1 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.24.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://172.24.0.2:9000", host: "loc.l:82"
2021/04/24 15:19:40 [error] 29#29: *1 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.24.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.24.0.2:9000", host: "loc.l:82", referrer: "http://loc.l:82/"
2021/04/24 15:56:08 [error] 29#29: *1 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.160.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.160.2:9000", host: "loc.l:83"
2021/04/24 15:56:08 [error] 29#29: *1 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.160.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://192.168.160.2:9000", host: "loc.l:83", referrer: "http://loc.l:83/"
2021/04/24 15:56:09 [error] 29#29: *1 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.160.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.160.2:9000", host: "loc.l:83"
2021/04/24 15:56:09 [error] 29#29: *1 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.160.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://192.168.160.2:9000", host: "loc.l:83", referrer: "http://loc.l:83/"
2021/04/24 16:01:33 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.176.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.176.2:9000", host: "loc.l:83"
2021/04/24 16:01:33 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.176.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://192.168.176.2:9000", host: "loc.l:83", referrer: "http://loc.l:83/"
2021/04/24 16:01:34 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.176.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.176.2:9000", host: "loc.l:83"
2021/04/24 16:01:34 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.176.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://192.168.176.2:9000", host: "loc.l:83", referrer: "http://loc.l:83/"
2021/04/24 16:09:15 [error] 28#28: *2 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.224.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.224.2:9000", host: "loc.l:83"
2021/04/24 16:09:15 [error] 28#28: *2 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.224.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://192.168.224.2:9000", host: "loc.l:83", referrer: "http://loc.l:83/"
2021/04/24 16:09:23 [error] 28#28: *2 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.224.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.224.2:9000", host: "loc.l:83"
2021/04/24 16:09:24 [error] 28#28: *2 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.224.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://192.168.224.2:9000", host: "loc.l:83", referrer: "http://loc.l:83/"
2021/04/24 16:10:24 [error] 28#28: *7 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.224.1, server: 127.0.0.1, request: "GET /_intellij_phpdebug_validator.php HTTP/1.1", upstream: "fastcgi://192.168.224.2:9000", host: "127.0.0.1:83"
2021/04/24 16:11:09 [error] 28#28: *9 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.224.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.224.2:9000", host: "loc.l:83"
2021/04/24 16:11:11 [error] 28#28: *9 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.224.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://192.168.224.2:9000", host: "loc.l:83", referrer: "http://loc.l:83/"
2021/04/24 16:11:21 [error] 28#28: *9 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.224.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.224.2:9000", host: "loc.l:83"
2021/04/24 16:11:22 [error] 28#28: *9 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.224.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://192.168.224.2:9000", host: "loc.l:83", referrer: "http://loc.l:83/"
2021/04/24 16:11:29 [error] 28#28: *9 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.224.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.224.2:9000", host: "loc.l:83"
2021/04/24 16:11:30 [error] 28#28: *9 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.224.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://192.168.224.2:9000", host: "loc.l:83", referrer: "http://loc.l:83/"
2021/04/24 16:13:00 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.240.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.240.2:9000", host: "loc.l:83"
2021/04/24 16:13:01 [error] 28#28: *4 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.240.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://192.168.240.2:9000", host: "loc.l:83", referrer: "http://loc.l:83/"
2021/04/24 16:13:22 [error] 28#28: *4 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.240.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.240.2:9000", host: "loc.l:83"
2021/04/24 16:13:23 [error] 28#28: *4 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.240.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://192.168.240.2:9000", host: "loc.l:83", referrer: "http://loc.l:83/"
2021/04/24 16:13:44 [error] 28#28: *4 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.240.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.240.2:9000", host: "loc.l:83"
2021/04/24 16:13:45 [error] 28#28: *4 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.240.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://192.168.240.2:9000", host: "loc.l:83", referrer: "http://loc.l:83/"
2021/04/24 16:13:55 [error] 28#28: *4 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.240.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.240.2:9000", host: "loc.l:83"
2021/04/24 16:13:56 [error] 28#28: *4 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.240.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://192.168.240.2:9000", host: "loc.l:83", referrer: "http://loc.l:83/"
2021/04/24 16:14:36 [error] 28#28: *4 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.240.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.240.2:9000", host: "loc.l:83"
2021/04/24 16:14:36 [error] 28#28: *4 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.240.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://192.168.240.2:9000", host: "loc.l:83", referrer: "http://loc.l:83/"
2021/04/24 16:14:37 [error] 28#28: *4 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.240.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.240.2:9000", host: "loc.l:83"
2021/04/24 16:14:37 [error] 28#28: *4 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.240.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://192.168.240.2:9000", host: "loc.l:83", referrer: "http://loc.l:83/"
2021/04/24 16:14:38 [error] 28#28: *4 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.240.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.240.2:9000", host: "loc.l:83"
2021/04/24 16:14:38 [error] 28#28: *4 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.240.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://192.168.240.2:9000", host: "loc.l:83", referrer: "http://loc.l:83/"
2021/04/24 16:14:44 [error] 28#28: *4 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.240.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.240.2:9000", host: "loc.l:83"
2021/04/24 16:14:46 [error] 28#28: *4 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.240.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://192.168.240.2:9000", host: "loc.l:83", referrer: "http://loc.l:83/"
2021/04/24 16:15:01 [error] 28#28: *4 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.240.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.240.2:9000", host: "loc.l:83"
2021/04/24 16:15:05 [error] 28#28: *4 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 192.168.240.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://192.168.240.2:9000", host: "loc.l:83", referrer: "http://loc.l:83/"
2021/04/24 16:15:53 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "loc.l:83"
2021/04/24 16:15:54 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.18.0.2:9000", host: "loc.l:83", referrer: "http://loc.l:83/"
2021/04/24 16:16:33 [error] 28#28: *2 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.19.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.2:9000", host: "loc.l:83"
2021/04/24 16:16:35 [error] 28#28: *2 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.19.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.19.0.2:9000", host: "loc.l:83", referrer: "http://loc.l:83/"
2021/04/24 16:18:32 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.20.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.2:9000", host: "loc.l:83"
2021/04/24 16:18:32 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.20.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.20.0.2:9000", host: "loc.l:83", referrer: "http://loc.l:83/"
2021/04/24 16:19:04 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.20.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.2:9000", host: "loc.l:83"
2021/04/24 16:20:00 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.20.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.20.0.2:9000", host: "loc.l:83", referrer: "http://loc.l:83/"
2021/04/24 16:20:39 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.20.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.2:9000", host: "loc.l:83"
2021/04/24 16:20:41 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.20.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.20.0.2:9000", host: "loc.l:83", referrer: "http://loc.l:83/"
2021/04/24 16:20:45 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.20.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.2:9000", host: "loc.l:83"
2021/04/24 16:20:46 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.20.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.20.0.2:9000", host: "loc.l:83", referrer: "http://loc.l:83/"
2021/04/24 16:21:47 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.20.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.2:9000", host: "loc.l:83"
2021/04/24 16:21:48 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.20.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.20.0.2:9000", host: "loc.l:83", referrer: "http://loc.l:83/"
2021/04/24 19:09:34 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.22.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://172.22.0.2:9000", host: "loc.l:83"
2021/04/24 19:09:35 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.22.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.22.0.2:9000", host: "loc.l:83", referrer: "http://loc.l:83/"
2021/04/24 19:10:46 [error] 28#28: *6 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.22.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://172.22.0.2:9000", host: "loc.l:83"
2021/04/24 19:10:47 [error] 28#28: *6 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.22.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.22.0.2:9000", host: "loc.l:83", referrer: "http://loc.l:83/"
2021/04/24 19:11:10 [error] 28#28: *6 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.22.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://172.22.0.2:9000", host: "loc.l:83"
2021/04/24 19:11:11 [error] 28#28: *6 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.22.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.22.0.2:9000", host: "loc.l:83", referrer: "http://loc.l:83/"
2021/04/24 19:11:47 [error] 28#28: *6 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.22.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://172.22.0.2:9000", host: "loc.l:83"
2021/04/24 19:11:47 [error] 28#28: *6 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.22.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.22.0.2:9000", host: "loc.l:83", referrer: "http://loc.l:83/"
2021/04/24 19:11:51 [error] 28#28: *6 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.22.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://172.22.0.2:9000", host: "loc.l:83"
2021/04/24 19:11:51 [error] 28#28: *6 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.22.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.22.0.2:9000", host: "loc.l:83", referrer: "http://loc.l:83/"
2021/04/24 19:14:20 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.23.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "fastcgi://172.23.0.2:9000", host: "loc.l:83"
2021/04/24 19:14:21 [error] 28#28: *1 FastCGI sent in stderr: "PHP message: Xdebug could not open the remote debug file '/tmp/xdebug.log'" while reading response header from upstream, client: 172.23.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.23.0.2:9000", host: "loc.l:83", referrer: "http://loc.l:83/"
